app/controllers/application_controller.rb

```diff
@@ -1,10 +1,88 @@
 require './config/environment'
+require 'pry'
 
 class ApplicationController < Sinatra::Base
```

Comment:
This controller is overloaded. Consider separating your concerns by splitting some of these actions into separate controllers (i.e. a users controller and a tweets controller, which you have but are not using).

Also, this is where your configurations are set, and all other controllers inherit from this one, so there is no need for the `register`, `set :views`, or `configure do #...` anywhere other than here.